src/drivers/char/terminal/vtparse.c

```diff
@@ -46,10 +46,11 @@ static void do_action(struct vtparse *parser, vt_action_t action, char ch) {
 		break;
 
 	case VT_ACTION_COLLECT:
-		if (token->attrs_len == -1) {
+		// Buffer overflow check may be needed
+		//if (token->attrs_len == -1) {
```

Comment:
@Kakadu May be in some platforms char can be unsigned? I offer to change type of **token->attrs_len** to **int** and remove comments